src/s2wasm.h

```diff
@@ -696,7 +700,7 @@ class S2WasmBuilder {
         Name target = cleanFunction(getCommaSeparated());
         auto aliased = aliasedFunctions.find(target);
         if (aliased != aliasedFunctions.end()) target = aliased->second;
-        if (implementedFunctions.count(target) > 0) {
+        if (implementedFunctions.count(target) != 0) {
```

Comment:
Oh extra change, I initially modified `implementedFunctions` and had to change all its uses, but then put it back. I didn't realize I'd changed it to something else (I was just restoring the old behavior), but it can only ever be 0 or 1 so `!=` seems more natural?